config/config.yml

```diff
@@ -3,3 +3,7 @@ rest:
   host: api7.dspace.org
   port: 443
   nameSpace: /server
+
+info:
+  enableEndUserAgreement: false
+  enablePrivacyStatement: false
```

Comment:
Why are these set to a default value of `false` in config.yml? Shouldn't they be `true` or just unspecified here (as they are already in `default-app-config.ts`)?

Reply:
These will indeed by default be true because `default-app-config.ts` I have removed this configuration.